.github/workflows/lint.yaml

```diff
@@ -28,20 +61,50 @@ jobs:
           cache: "npm"
           cache-dependency-path: test/aidbox_sdk/snapshots/typescript/package-lock.json
 
-      # - name: Cache Node modules
-      #   uses: actions/cache@v3
-      #   with:
-      #     path: |
-      #       ~/.npm
-      #       **/node_modules
-      #     key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-      #     restore-keys: |
-      #       ${{ runner.os }}-node-
-
       - name: Install system and client dependencies
         run: |
           sudo apt -y update
           cd test/aidbox_sdk/snapshots/typescript && npm ci
 
       - name: Run ESLint
         run: cd test/aidbox_sdk/snapshots/typescript && npm run lint
+
+  lint-python:
+    needs: detect-changes
+    if: ${{ needs.detect-changes.outputs.python-changed == 'true' }}
```

Comment:
It looks like `${{ needs.detect-changes.outputs.python-changed == 'true' }}` can be simplified to `${{ needs.detect-changes.outputs.python-changed }}`